src/components/molecules/CreateToken/CreateToken.tsx

```diff
@@ -24,30 +29,46 @@ interface CreateTokenProps {
 }
 
 export const CreateToken = (props: CreateTokenProps) => (
-  <div className={'CreateToken'}>
-    <header className={'CreateToken__header'}>
-      <h2 className={'CreateToken__header__title'}>API Tokens</h2>
-      <p className={'CreateToken__header__description'}>
-        Manage the ways that you authorize requests to the Frost API.
-      </p>
-    </header>
-    <BoxToken apiTokens={props.boxToken} onDeleteToken={props.onShowModal} />
-    <LegendVerifiedAccount
-      show={!props.showVerifiedAccount}
-      onClick={props.sendEmailVarifiedAccount}
-      retryWait={props.retryWait}
-    />
-    <Button
-      className={'CreateToken__button'}
-      text={props.textCreateTokenButton}
-      onClick={props.onCreateApiToken}
-      disabled={props.submitDisabled}
-    />
-    <DeleteToken
-      onDeleteToken={props.onDeleteToken}
-      show={props.showDeleteModal}
-      onClose={props.onCloseModal}
-      disabledButton={props.disabledButton}
-    />
+  <div className={'CreateTokenContainer'}>
+    <div className={'CreateToken'}>
+      <header className={'CreateTokenContainer__CreateToken__header'}>
+        <h2 className={'CreateTokenContainer__CreateToken__header__title'}>API Tokens</h2>
+        <p className={'CreateTokenContainer__CreateToken__header__description'}>
+          Manage your API tokens for authenticating with the Frost API.
+        </p>
+      </header>
+      <BoxToken apiTokens={props.boxToken} onDeleteToken={props.onShowModal} />
+      <LegendVerifiedAccount
+        show={!props.showVerifiedAccount}
+        onClick={props.sendEmailVarifiedAccount}
+        retryWait={props.retryWait}
+      />
+      <Button
+        className={'CreateTokenContainer__CreateToken__button'}
+        text={props.textCreateTokenButton}
+        onClick={props.onCreateApiToken}
+        disabled={props.submitDisabled}
+      />
+      <DeleteToken
+        onDeleteToken={props.onDeleteToken}
+        show={props.showDeleteModal}
+        onClose={props.onCloseModal}
+        disabledButton={props.disabledButton}
+      />
+    </div>
+    {isActiveFeatureName(FeatureName.RunKit, getCurrentActiveFeatureNames({ initialFeatures })) &&
```

Comment:
How come we are not using the ReactFeatureToggles Component? Is it because its setup incorrectly still?

Reply:
I didn't think it needed a fallback, so I thought this would be better. I can change it though.

Reply:
inactiveComponent of Feature can render nothing by returning null. `() => null`.

The way you have done this currently requires knowing about where to import the features instead of allowing the root of the application provide the features through react context.